src/Cli.Tests/InitTests.cs

```diff
@@ -187,14 +192,40 @@ public void CosmosDbNoSqlDatabase()
             RunTest(options, expectedRuntimeConfig);
         }
 
+        /// <summary>
+        /// Verify that if graphQLSchema file is not present, we will get error.
```

Comment:
Is there a specific error message associated with the failure? if so can you update summary to say "Verify that if graphQLSchema file is not present, the CLI tool returns the error "Error message/Error code"